### PR TITLE
Rename Platform and Get Started

### PR DESCRIPTION
**Pull request by journey2w**
### Change Description ###

Rename from 'Open Source Tizen' to 'Platform' and from 'Getting Started' to 'Get Started'

### CODE REVIEW

File: docs/platform/tizen-rt/public-api/index.html [hunk 1]
```diff
@@ -110,8 +110,8 @@
 <p>For Additional information of Tizen RT, please see below Documentation.</p>
 <h3>Documentation</h3>
 <ul>
-<li>Tizen RT Getting Started <a href="https://source.tizen.org/documentation/tizen-rt/tizen-rt-getting-started">page</a></li>
-<li>Tizen RT 1.0 Specification <a href="https://source.tizen.org/documentation/tizen-rt/tizen-rt-1.0-specification">page</a> </li>
+<li>Tizen RT Getting Started <a href="/platform/tizen-rt/rt-getting-started">page</a></li>
```
Comment (mijin85cho) [inline docs/platform/tizen-rt/public-api/index.html]: I'm not sure with update as the hierarchy doesn't match with folder structure. Would you please check?

**Comment by journey2w** [comment]
In order to rebuild, I'll close this PR and reopen it.

File: docs/platform/tizen-studio/meta-package.md [hunk 2]
```diff
@@ -66,4 +66,4 @@ Write the following additional information for extension meta packages:
            - Deployed as a repository of all extensions (Repo Server, Image, Package)
            ```
 
-For more information, see [Configuring the Extension SDK Repository](https://developer.tizen.org/development/tizen-studio/download/configuring-package-manager#extension).
+For more information, see [Configuring the Extension SDK Repository](../../application/tizen-studio/setup/advanced-configuration#configuring-the-extension-sdk-repository).
```
Comment (mijin85cho) [inline docs/platform/tizen-studio/meta-package.md]: Broken link. Please add `.md`. "advanced-configuration.md"

File: docs/platform/tizen-studio/overview.md [hunk 3]
```diff
@@ -35,8 +35,8 @@ If you have found a bug, have any questions, or want to contribute, feel free to
 
 - [Tizen Studio Download](https://developer.tizen.org/development/tizen-studio/download)
 - [OpenSource Repository](https://review.tizen.org/gerrit/#/)
-- [Tool Guide](https://developer.tizen.org/development/tizen-studio/native-tools)
-- [Training](https://developer.tizen.org/development/training/overview)
-- [API References](https://developer.tizen.org/development/api-references/native-application)
-- [Release Notes](https://developer.tizen.org/development/tizen-studio/download/release-notes)
+- [Tool Guide](../../application/tizen-studio/native-tools/index.md)
+- [Get Started for Native Application](../../application/native/get-started/overview.md)
+- [Get Started for Web Application](../../application/native/get-started/overview.md)
```
Comment (mijin85cho) [inline docs/platform/tizen-studio/overview.md]: This link to Web is actually going to native.

File: docs/platform/tizen-studio/local-build.md [hunk 4]
```diff
@@ -72,4 +72,4 @@ To build the code locally for testing:
 
 6. Install the package with the Package Manager.
 
-   For more information on the Package Manager, see [Configuring the Extension SDK Repository](https://developer.tizen.org/development/tizen-studio/download/configuring-package-manager#extension).
+   For more information on the Package Manager, see [Configuring the Extension SDK Repository](../../application/tizen-studio/setup/advanced-configuration#configuring-the-extension-sdk-repository).
```
[the remark this answers is not in the document]
Comment (journey2w) [reply docs/platform/tizen-studio/local-build.md]: Done.

File: docs/platform/tizen-studio/overview.md (hunk 3 above)
Comment (journey2w) [reply docs/platform/tizen-studio/overview.md]: Done.

File: docs/platform/tizen-studio/meta-package.md (hunk 2 above)
Comment (journey2w) [reply docs/platform/tizen-studio/meta-package.md]: Done.

**Pull request merged**